camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -938,6 +939,22 @@ def pt2line(x0, y0, x1, y1, x2, y2):
         else:
             return None
 
+    def update_mask(self, img, screen):
+        """
+        Update or if None create a red mask with the same size of screen
+        and add on top of it detection areas displayed through green circles
+        """
+        height, width = img.shape[0], img.shape[1]
+        shape = screen.shape
+        scale_height, scale_width = height / shape[0], width / shape[1]
+        if self.mask is None:
+            self.mask = numpy.uint8(numpy.ones((shape[0], shape[1], 1)) * (0, 0, 255))
+        else:
+            for corners in self.good_corners:
+                downsampled_corners = [
+                    [int(corner[0][0] / scale_height), int(corner[0][1] / scale_width)] for corner in corners[0]]
+                for index, el in enumerate(downsampled_corners):
```

Comment:
index is not used, call it _ , and el is too short variable name, please call it element